django/db/backends/postgresql/base.py

```diff
@@ -365,17 +440,81 @@ def _nodb_cursor(self):
     @cached_property
     def pg_version(self):
         with self.temporary_connection():
-            return self.connection.server_version
+            return self.connection.info.server_version
 
     def make_debug_cursor(self, cursor):
         return CursorDebugWrapper(cursor, self)
 
 
-class CursorDebugWrapper(BaseCursorDebugWrapper):
-    def copy_expert(self, sql, file, *args):
```

Comment:
Should we check whether we can provide `copy_expert` and `copy_to` via a new `copy` method like it will be used on `pyscopg3` and deprecate the old ones?

More importantly, those two APIs don't seem to be part of the public API (not documented), so we could just drop them?

Reply:
> Should we check whether we can provide `copy_expert` and `copy_to` via a new `copy` method like it will be used on `pyscopg3` and deprecate the old ones?
> 
> More importantly, those two APIs don't seem to be part of the public API (not documented), so we could just drop them?

`copy_to` and `copy_expert` are `psycopg2.cursor` methods, we only log their execution, so there is no need to provide their equivalents on `psycopg` version 3.  We can drop them.